package.json

```diff
@@ -27,7 +27,6 @@
     "build-es5": "mkdir -p dist && babel src --out-dir dist --plugins=transform-es2015-modules-commonjs --source-maps inline",
     "build-es6": "mkdir -p dist-es6 && babel src --out-dir dist-es6 --source-maps inline",
     "clean": "rm -fr dist/* dist-es6/*",
-    "clean": "rm -fr dist/* dist-es6/*",
```

Comment:
it was in there twice...but I see it was removed by last commit https://github.com/uber/react-map-gl/commit/6e3a934613a8711728c929cb85558b1ada073c6f